Library/Formula/bazel.rb

```diff
@@ -0,0 +1,25 @@
+class Bazel < Formula
+  homepage "http://bazel.io/"
+  url "https://github.com/google/bazel.git", :tag => "0.0.2"
+  sha256 "5865356e74622c248248174acd09f38c07f3983867d7e687fa5932f2216e84c0"
+
+  head "https://github.com/google/bazel.git"
+
+  depends_on "protobuf" => :build
+  depends_on "libarchive"
+
```

Comment:
We have a [special java dep mechanism](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/java_dependency.rb).